### ISSUE TITLE
Hykuup does not build on Github due to rails issue

### ISSUE THREAD

**Issue by laritakr** [scientist-softserv/hykuup_knapsack#175]
Refs: https://github.com/scientist-softserv/hykuup_knapsack/pull/174

- [ ] Fix [this error](https://github.com/scientist-softserv/hykuup_knapsack/actions/runs/8288780977/job/22683973052?pr=174)

**Comment by ShanaLMoore** [comment]
I'm going to take a swing at this but time box it to an hr @bkiahstroud

**Comment by ShanaLMoore** [comment]
related: https://github.com/samvera/hyku/issues/2155

**Comment by ShanaLMoore** [comment]
built and tried to push up an image. I receive this error: 

```ruby
unauthorized: unauthenticated: User cannot be authenticated with the token provided.
```

**Comment by bkiahstroud** [comment]
> built and tried to push up an image. I receive this error, even after docker login:

Did you try logging into the registry? `docker login ghcr.io`

**Comment by bkiahstroud** [comment]
Latest error ([run](https://github.com/scientist-softserv/hykuup_knapsack/actions/runs/8396791500/job/22998904132)): 

```
#13 2180.9 <jemalloc>: MADV_DONTNEED does not work (memset will be used instead)
#13 2180.9 <jemalloc>: (This is the expected behaviour if you are running under QEMU)
#13 2180.9 gumbo.c:32:10: fatal error: gumbo.h: No such file or directory
#13 2180.9    32 | #include "gumbo.h"
#13 2180.9       |          ^~~~~~~~~
#13 2180.9 compilation terminated.
#13 2180.9 make: *** [Makefile:472: gumbo.o] Error 1
#13 2180.9 
#13 2180.9 make failed, exit code 2
#13 2180.9 
#13 2180.9 Gem files will remain installed in /usr/local/bundle/gems/nokogiri-1.13.10 for
#13 2180.9 inspection.
#13 2180.9 Results logged to
#13 2180.9 /usr/local/bundle/extensions/aarch64-linux-musl/3.2.0/nokogiri-1.13.10/gem_make.out
...
#13 2180.9 An error occurred while installing nokogiri (1.13.10), and Bundler cannot
#13 2180.9 continue.
#13 2180.9 
#13 2180.9 In Gemfile:
#13 2180.9   active_elastic_job was resolved to 2.0.1, which depends on
#13 2180.9     rails was resolved to 6.1.7.6, which depends on
#13 2180.9       actioncable was resolved to 6.1.7.6, which depends on
#13 2180.9         actionpack was resolved to 6.1.7.6, which depends on
#13 2180.9           actionview was resolved to 6.1.7.6, which depends on
#13 2180.9             rails-dom-testing was resolved to 2.2.0, which depends on
#13 2180.9               nokogiri
#13 ERROR: process "/bin/sh -c git config --global --add safe.directory /app/samvera &&   bundle install --jobs \"$(nproc)\"" did not complete successfully: exit code: 5
```

**Comment by ShanaLMoore** [comment]
Fixed. 

ref: https://assaydepot.slack.com/archives/C0313NKG2DA/p1711060976609249
